jobserver/settings.py

```diff
@@ -45,6 +45,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env.bool("DEBUG", default=False)
 
+DEBUG_TOOLBAR = env.bool("DJANGO_DEBUG_TOOLBAR", default=False)
```

Comment:
In Airlock, we do the following:

```py
if DEBUG:  # pragma: no cover
    DJANGO_DEBUG_TOOLBAR = os.environ.get("DJANGO_DEBUG_TOOLBAR", "false") == "True"
else:  # pragma: no cover
    DJANGO_DEBUG_TOOLBAR = False
```

@evansd, you mentioned that we should avoid making lots of behaviour conditional on `DEBUG`. For this reason, I've not done in job-server what we do in Airlock. Have I missed something?

Reply:
No, I think you're right.

The [default behaviour](https://django-debug-toolbar.readthedocs.io/en/stable/configuration.html#show-toolbar-callback) of DDT is to disable itself when `DEBUG = False` so in one sense there's not much point allowing it to be enabled when `DEBUG = False`. But I think it's better to keep things independent.